yt_dlp/extractor/archiveorg.py

```diff
@@ -241,3 +251,87 @@ def _real_extract(self, url):
                     'parent': 'root'})
 
         return info
+
+
+class YoutubeWebArchiveIE(InfoExtractor):
+    IE_NAME = 'youtube:web.archive.org'
```

Comment:
should probably be `archive.org:something` (is web.archive.org and www.archive.org different things?)

Reply:
web.archive.org is the Wayback Machine, part of archive.org.

Reply:
so, `web.archive:youtube` maybe?